components/brave_wallet/browser/brave_wallet_provider_impl.h

```diff
@@ -43,21 +48,31 @@ class BraveWalletProviderImpl final
                                     const std::vector<std::string>& accounts);
   void GetChainId(GetChainIdCallback callback) override;
   void GetAllowedAccounts(GetAllowedAccountsCallback callback) override;
+  void AddEthereumChain(const std::string& json_payload,
+                        AddEthereumChainCallback callback) override;
+  void OnAddEthereumChain(const std::string& chain_id, bool accepted);
```

Comment:
move this to the private section pls and thx

Reply:
done